packages/mobile/src/components/ContactCircle.v2.tsx

```diff
@@ -0,0 +1,92 @@
+import colors from '@celo/react-components/styles/colors'
```

Comment:
Sounds good

Reply:
Actually, took a 2nd look and I am a hesitant to replace because the original `ContactCircle` is in the higher-level `react-components` directory. The fact that we use both react directories interchangeably has been confusing for me and gives me pause with regards to making changes. Would be great if we settled on one place for mobile components and imported from web into a mobile components directory as needed.

Are you sure that `ContactCircle` and `Avatar` are exclusively used in the`mobile` directory? Looks like it's also used in `verifier` but I am not sure if that's actually mobile related.

Reply:
verifier was an app for forwarding text messages. prior to attestation service. 

react-components is only used by mobile and verifier